pyproject.toml

```
@@ -1,3 +1,47 @@
+[project]
+name = "readme_renderer"
+version = "41.0"
+description = "readme_renderer is a library for rendering readme descriptions for Warehouse"
+authors = [
+  {name = "The Python Packaging Authority", email = "admin@mail.pypi.org"}
+]
+readme = "README.rst"
+license = {file = "LICENSE"}
```

This directive changes the metadata to **embed** the license text into the metadata, previously this would be a string of the license type, not the entire file.
This change would preserve the value from `__about__.py`:
```suggestion
license = {text = "Apache License, Version 2.0"}
```